tests/Microsoft.Identity.Test.Unit/PublicApiTests/LoggerTests.cs

```diff
@@ -239,6 +239,81 @@ public async Task IdentityLoggerOverridesLegacyLoggerTestAsync()
             }
         }
 
+        [TestMethod]
+        [DataRow(false)]
+        [DataRow(true)]
+        public async Task ExternalMsalLoggerTestAsync(bool piiLogging)
```

Comment:
Can you add or identify a test for the legacy logger? Just to see that we can still log both message with pii and without pii.